examples/geoflow/1a63.in

```diff
@@ -2,13 +2,14 @@ read
     mol pqr 1a63.pqr
 end
 elec name comp_solv        # Solvated complex
-    geoflow
+    geoflow-auto
```

Comment:
What does `-auto` mean here?  Does the older syntax still work as well?

Reply:
Similar situation as the `ekey glob` comment. The [geoflow-auto](https://apbs.readthedocs.io/en/latest/using/input/elec/geoflow-auto.html) specification uses `geoflow-auto` as the keyword. I can implement the same approach to `ekey glob` by depricating  the `geoflow` keyword or implementing a re-write grammar.

Reply:
I am adding `geoflow` and `geoflow-auto` as keywords. This will allow the current code to run and the parser to still work. I am adding a `check_depricated` function to WARN the use and point out the keyword that should be changed.